lib/services/nodes-api-service.js

```diff
@@ -639,5 +639,39 @@ function nodeApiServiceFactory(
         });
     };
 
+    /**
+     * Get a list of all OBMs for the specified Node id
+     * @param  {String}     id
+     * @return {Promise}    Resolves to an array of OBMs
+     */
+    NodeApiService.prototype.getObmsByNodeId = function(id) {
+        return waterline.nodes.getNodeById(id)
+            .then(function (node){
+                if (!node) {
+                    throw new Errors.NotFoundError(
+                        'Node not Found ' + id
+                    );
+                }
+                return waterline.obms.findAllByNode(id, false);
+            });
+    };
+
+    /**
+     * Create an OBM for the specified Node id
+     * @param  {String}     id, obm
+     * @return {Promise}
+     */
+    NodeApiService.prototype.putObmsByNodeId = function(id, obm) {
+        return waterline.nodes.getNodeById(id)
```

Comment:
@geoff-reid You could replace getNodeById with needByIdentifier which will throw a NotFoundError if no node exists.